pkg/proxy/nftables/proxier.go

```diff
@@ -265,6 +264,52 @@ func NewProxier(ipFamily v1.IPFamily,
 	return proxier, nil
 }
 
+// Check if we can use the nftables proxy mode.
+func checkNFTables(ipFamily v1.IPFamily, allowOlderKernel bool) (knftables.Interface, error) {
```

Comment:
Prefer checkNFTablesProxyMode if that is what is being done..

Reply:
"nftables proxy mode" is implied by `pkg/proxy/nftables`.

The intent is that it's checking if the nftables client/kernel support is good enough for kube-proxy. Maybe `checkHostNFTables`...